src/init.cpp

```diff
@@ -837,12 +837,15 @@ static void StartupNotify(const ArgsManager& args)
 }
 #endif
 
-static void PeriodicStats(ArgsManager& args, ChainstateManager& chainman, const CTxMemPool& mempool)
+static void PeriodicStats(NodeContext& node)
 {
     assert(::g_stats_client->active());
+    const ArgsManager& args = *Assert(node.args);
+    ChainstateManager& chainman = *Assert(node.chainman);
```

Comment:
should not it be `const ChainstateManager& chainman = EnsureChainman(node);` ans same for other?

Reply:
`Ensure*` helpers are for RPCs